File: Class_member/2.rb
class Cs
    def Cs.class_method()
        p "Class method"
    end

    def instance_method()
        p "Instance method"
    end
end

i = Cs.new()
Cs.class_method()   # 클래스 메소드는 클래스의 소속으로 사용
                    # 즉, i.class_method()로는 실행 안 됨
i.instance_method() # 인스턴스 메소드는 인스턴스 소속으로 사용
                    # 즉, Cs.instance_method()로는 실행 안 됨

File: Class_member/2.py
class Cs(object):
    @staticmethod   # 클래스 메소드 멤버 설정을 위한 장식자
    def static_method():
        print("Static method")
    @classmethod    # 클래스 메소드 멤버 설정을 위한 장식자
    def class_method(cls):  # cls가 꼭 들어가야 함
        print("Class method")
    def instance_method(self):
        print("Instance method")

i = Cs()
Cs.static_method()  # 클래스 메소드
Cs.class_method()   # 클래스 메소드
i.instance_method()